pyamg/dev/elasticity_bar.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import print_function
```

Comment:
Don't know why these ended up in my complexity branch, these are for testing other more active development branches. Will probably remove from this, unless you think they should be kept.